src/com/missafternoon/budjen/Application.java

```diff
@@ -1,3 +1,7 @@
+package com.missafternoon.budjen;
+
+import com.missafternoon.budjen.transactions.*;
```

Comment:
I'm OK with importing `*` when talking about static methods on a class, e.g. `Assert.assertWhatever` methods, but I don't think we need to import every class in the `transactions` package in this class. Just import the ones you need instead. I'm guessing IntelliJ can help you out here.

This comment applies in multiple locations for this PR.